client/app/bundles/shared/components/Input.scss

```diff
@@ -0,0 +1,54 @@
+@import "base.scss";
+
+.label {
+  font-family: $font-family;
+  padding: 12px 0;
+  letter-spacing: 0.0944444em;
```

Comment:
We should use px values here. Thoughts @nshki?

Reply:
I think `em`s are okay here, mainly because that would ensure `letter-spacing` scales with font size changes.